devops/bin/update_server_list.bash

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+source /Users/bwu2/.github/token.sh
```

Comment:
parameterize it please.

Reply:
hmm, right, it sounds like a better design. will do.